elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -888,6 +888,11 @@ elytron.filesystem-realm.hash-encoding=The string format for the password if it
 elytron.filesystem-realm.hash-charset=The character set to use when converting the password string to a byte array.
 elytron.filesystem-realm.credential-store=The reference to the credential store that contains the secret key to encrypt and decrypt the realm.
 elytron.filesystem-realm.secret-key=The alias of the secret key to encrypt and decrypt the realm.
+elytron.filesystem-realm.key-store=The reference to the key store that contains the key pair to use to verify integrity.
+elytron.filesystem-realm.key-store-alias=The alias that identifies the PrivateKeyEntry within the key store to use to verify integrity.
+# Operations
+elytron.filesystem-realm.update-key-pair=Change the key pair associated with the filesystem realm for verifying integrity.
```

Comment:
> Change the key pair

The key pair can be changed using `write-attribute`. This operation rather updates the realm data (all the identities, in the actual implementation) to reflect the associated key pair.

Reply:
Updated to
`elytron.filesystem-realm.update-key-pair=Updates the filesystem realm to make use of the new key pair to verify integrity.
`